css/styles.css

```diff
@@ -57,6 +57,13 @@ a {
     margin: 10px auto 0;
 }
 
+#toggleLoopLabel{
```

Comment:
Please conform to the CSS formatting we're already using, that's why Travis CI failed the build. You can click the details link for the specific errors